### PR TITLE
Update for large disks

### PR DESCRIPTION
**Pull request by yuemlu**
support up to 32 TB

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     stored in an Azure storage account. Virtual machines also can have one or more data disks, that are also stored as VHDs.
 
 
-    Azure Managed and Unmanaged Data Disks have a maximum size of 4095 GB (with the exception of larger disks in preview). Azure Unmanaged Disks also have a maximum capacity of 4095 GB.
+    Azure Managed Data Disks have a maximum size of 32767 GB, and Azure Unmanaged Disks have a maximum capacity of 4095 GB.
```
Comment (adewaleo) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]:
```suggestion
    Azure Managed Data Disks have a maximum size of 32767 GB and Azure Unmanaged Disks have a maximum capacity of 4095 GB.
```

Just a suggestion

**Comment by yuemlu** [comment]
#sign-off

**Comment by yuemlu** [comment]
@adewaleo Addressed your comment and updated the doc for unmanaged disk in a different pull request

**Comment by adewaleo** [comment]
> @adewaleo Addressed your comment and updated the doc for unmanaged disk in a different pull request

@yuemlu could you also make the change you made to this [this line ](https://github.com/Azure/azure-cli/blob/82685556ec162d947f20a5b5f37fc1155c8cb239/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py#L626) and [this line](https://github.com/Azure/azure-cli/blob/82685556ec162d947f20a5b5f37fc1155c8cb239/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py#L1287)?

You made changes for the large description of `az disk`, but the linked help is for `az vm disk` and `az vm unmanaged disk`

**Comment by adewaleo** [comment]
@yuemlu, have you seen my latest comments?

**Comment by yuemlu** [comment]
@adewaleo What other changes do you need? I believe I have addressed all.

**Comment by adewaleo** [comment]
@yuemlu have you addressed the feedback in [this comment](https://github.com/Azure/azure-cli/pull/8858#issuecomment-476030835)?

There are other long summaries that make mention of the old maximum size.

**Comment by tjprescott** [comment]
@yuemlu please resolve the review comments or this PR will be closed due to inactivity.

**Comment by tjprescott** [comment]
Closing due to inactivity. Feel free to address the comments and reopen.

**Pull request closed without merging**